cmd/ci-operator/main.go

```diff
@@ -450,16 +450,21 @@ func (o *options) Complete() error {
 		o.templates = append(o.templates, template)
 	}
 
-	clusterConfig, err := util.LoadClusterConfig()
-	if err != nil {
-		return fmt.Errorf("failed to load cluster config: %v", err)
-	}
+	o.clusterConfig = &rest.Config{}
```

Comment:
I know in principle, it should be fine if we use `nil`.
However, we use this config to init clients.

https://github.com/openshift/ci-tools/blob/a0e763cba29a22c2b09fd51236c991a1fb2fbcb5/pkg/defaults/defaults.go#L68-L79

And we usually do not check if the client is nil before using it.
Really not easy with nil clusterConfig.